internals/detect_intent.py

```diff
@@ -65,6 +67,10 @@ def detect_field(subject):
 CHROMESTATUS_LINK_ALTERNATE_RE = re.compile(
     r'entry on the feature dashboard:?\s+'
     r'https://www.chromestatus.com/feature/(\d+)', re.I)
+NOT_LGTM_RE = re.compile(
+    r'\b(not|almost|need|want|missing) (a |an )?LGTM\b',
```

Comment:
I thought it might come up in a sentence, like "You are still missing an LGTM" which should not be counted as an LGTM.  This is all kind of speculative.  In looking at the messages in the blink-dev archive, there are no examples of anything like these so far, but I imagine that they might be more useful to have than to not have.